packages/hdwallet-keepkey/src/cosmos.ts

```diff
@@ -202,6 +203,13 @@ export async function cosmosSignTx(transport: Transport, msg: core.CosmosSignTx)
         };
       },
     };
-    return await (await protoTxBuilder).sign(msg.tx, offlineSigner, msg.sequence, msg.account_number, msg.chain_id);
+
+    const signerData: SignerData = {
+      sequence: Number(msg.sequence),
+      accountNumber: Number(msg.account_number),
+      chainId: msg.chain_id,
+    };
+
+    return (await protoTxBuilder).sign(address, msg.tx as StdTx, offlineSigner, signerData);
```

Comment:
compiler isn't smart enough to narrow a `core.ThorchainStdTx` -> `cosmjs.amino.StdTx` and know that it isn't a `ProtoTx`. I could recreate a `StdTx` type and get rid of the as possibly